### PR TITLE
Add reviews table and form

### PR DESCRIPTION
**Pull request by afiakyalo**
- Added reviews table 
- Added reviews to park showpages
- Added controller tests and enzyme tests

### CODE REVIEW

File: app/javascript/react/components/ParkShowContainer.js [hunk 1]
```diff
@@ -19,15 +56,40 @@ const ParksShowContainer = props => {
...
+          error = new Error(errorMessage)
+          throw error
+        }
+      })
+      .then(response => response.json())
+      .then(reviewData => {
+        setReviews(reviewData.reviews)
       })
       .catch(error => console.error(`Error in fetch: ${error.message}`));
     }, [])
```
Comment (nwalberts) [inline app/javascript/react/components/ParkShowContainer.js]: there shouldnt be two fetch requests for info here. this one should be merged with the first one. I may merge this in just so you can move forward, but then would ask you to refactor this fetch request into a combined fetch in the future.

File: app/javascript/react/components/ParkShowContainer.js [hunk 2]
```diff
@@ -36,12 +98,69 @@ const ParksShowContainer = props => {
         rating={parkInfo.rating}
         description={parkInfo.description}
         photo={parkInfo.photo}
+        addNewReview={addNewReview}
       />
     }
```
Comment (nwalberts) [inline app/javascript/react/components/ParkShowContainer.js]: in what circumstance would `parkInfo` not be truthy? lets just move this into the return

File: app/javascript/react/components/ReviewTile.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from "react"
+
+import ParkShowContainer from "./ParkShowContainer"
```
Comment (nwalberts) [inline app/javascript/react/components/ReviewTile.js]: Ive no idea why this is being imported here?

File: app/serializers/park_show_serializer.rb [hunk 4]
```diff
@@ -1,3 +1,5 @@
 class ParkShowSerializer < ActiveModel::Serializer
   attributes :id, :name, :city, :state, :zip, :rating, :description, :photo
+
+  has_many :reviews
 end
```
Comment (nwalberts) [inline app/serializers/park_show_serializer.rb]: these two serializers look like they do nearly the exact same thing. We should be merging them into one serializer

File: app/javascript/react/components/ParkShowContainer.js (hunk 1 above)
Comment (hhorobin) [reply app/javascript/react/components/ParkShowContainer.js]: pending

**Pull request merged**